src/coffee/fetchstocks.coffee

```diff
@@ -8,7 +8,8 @@ class FetchStocks
 
 
   run: ->
-    query = @client.inventoryEntries.all().sort('sku').expand('supplyChannel')
+    # Do NOT use custom sorting due to https://github.com/sphereio/sphere-node-sdk/issues/194
```

Comment:
Nobody will notice this warning in commits or PR text. It is there to prevent somebody adding this "feature" again.